geo/src/main/java/org/odk/collect/geo/selection/SelectionMapFragment.kt

```diff
@@ -218,15 +220,15 @@ class SelectionMapFragment(
         summarySheetBehavior = BottomSheetBehavior.from(summarySheet)
         summarySheetBehavior.state = STATE_HIDDEN
 
-        val onBackPressedCallback = object : OnBackPressedCallback(false) {
+        val closeSummarySheet = object : OnBackPressedCallback(false) {
             override fun handleOnBackPressed() {
                 summarySheetBehavior.state = STATE_HIDDEN
             }
         }
 
-        requireActivity().onBackPressedDispatcher.addCallback(
+        (onBackPressedDispatcher ?: requireActivity().onBackPressedDispatcher).addCallback(
```

Comment:
To my mind, using the host `Activity` `OnBackPressedDispatcher` means we only need to test this is configured correctly when we override the value (like in `SelectOneFromMapDialogFragment`).